Vagrantfile

```diff
@@ -10,16 +10,17 @@ Vagrant.configure("2") do |config|
 
   # Port forwarding. The third argument only allows access via 127.0.0.1 to
   # disable public access.
-  #config.vm.network "forwarded_port", guest: 80, host: 8080, host_ip: "127.0.0.1"
+  config.vm.network "forwarded_port", guest: 31811, host: 8080, host_ip: "127.0.0.1"
```

Comment:
This allows to test changes locally by typing `http://localhost:8080` in your browser.